DQM/SiPixelMonitorClient/interface/SiPixelActionExecutor.h

```diff
@@ -84,7 +84,8 @@ class SiPixelActionExecutor {
  bool readConfiguration(	    int 			 & tkmap_freq, 
                         	    int 			 & summary_freq);
  void readConfiguration(	    );
-
+ int getLadder(                     std::string                    dname_);
```

Comment:
@minano - Two small changes - 1) please make this a const std::string & instead of a std::string. 2) usually variables ending with '_' are for member data not method specific variables. So could you remove the '_' in dname_.